### PR TITLE
Add workflow for uploading release to PyPI

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
Token has been added to the project secrets.

Signed-off-by: Frantisek Lachman <flachman@redhat.com>

### CODE REVIEW

File: .github/workflows/pypi-publish.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        # https://pypa-build.readthedocs.io
+        run: |
+          python -m pip install build
+          python -m build --sdist --wheel
+
+      - name: Publish 📦 to PyPI
+        # https://github.com/pypa/gh-action-pypi-publish
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          # secrets.PYPI_API_TOKEN is set to usercont-release-bot user token
```
Comment (jpopelka) [inline .github/workflows/pypi-publish.yml:28]: is this still true, or is it set to your token?

**Review by jpopelka — APPROVED** [review]
I've just scaled down the [release-bot instance for colin](https://open.paas.redhat.com/console/project/jkosciel-release-bot/browse/dc/colin-release).

File: .github/workflows/pypi-publish.yml (hunk 1 above)
Comment (lachmanfrantisek) [reply .github/workflows/pypi-publish.yml:28]: I've used the `usercont-release-bot` account to generate it.

Comment (jpopelka) [reply .github/workflows/pypi-publish.yml:28]: ok, then feel free to merge

**Review by lslebodn — APPROVED** [review]
+1

**Pull request merged**